Userland/Libraries/LibJS/Runtime/Temporal/PlainDatePrototype.cpp

```diff
@@ -493,18 +493,18 @@ JS_DEFINE_NATIVE_FUNCTION(PlainDatePrototype::until)
 
     // 15. If smallestUnit is not "day" or roundingIncrement ≠ 1, then
     if (*smallest_unit != "day"sv || rounding_increment != 1) {
-        // a. Set result to ? RoundDuration(result.[[Years]], result.[[Months]], result.[[Weeks]], result.[[Days]], 0, 0, 0, 0, 0, 0, roundingIncrement, smallestUnit, roundingMode, temporalDate).
+        // a. Set result to (? RoundDuration(result.[[Years]], result.[[Months]], result.[[Weeks]], result.[[Days]], 0, 0, 0, 0, 0, 0, roundingIncrement, smallestUnit, roundingMode, temporalDate)).[[DurationRecord]].
         // See NOTE above about why this is done.
```

Comment:
Because I can't comment on the NOTE because it's in a hidden section:
The NOTE above should now say the type is `DurationRecord`.
Better yet, the changes in `since` can also be applied here.

Reply:
>  Better yet, the changes in since can also be applied here.

Ah, that's so much better! :^)

cc @IdanHo, the description in the old code is basically answering your question above.